.github/workflows/ci.yml

```diff
@@ -145,8 +134,70 @@ jobs:
       - name: Test
         run: ./phpunit
         shell: bash
+
+  test-reset-database-with-migration:
+    name: Test migration - D:${{ matrix.database }}, ${{ matrix.use-dama == 1 && ' (dama)' || '' }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        database: [ mysql, pgsql, sqlite ]
+        use-dama: [ 0, 1 ]
+        with-migration-configuration-file:
+          - ''
+          - 'tests/Fixture/MigrationTests/configs/migration-configuration.php'
+          - 'tests/Fixture/MigrationTests/configs/migration-configuration-transactional.php'
+        exclude:
+          - database: sqlite
+            use-dama: 0
+          - database: mysql
+            with-migration-configuration-file: 'tests/Fixture/MigrationTests/configs/migration-configuration-transactional.php'
```

Comment:
I also discovered that for an obscure reason, transactional mode + MySQL is buggy - at least in our tests, I'm not using MySQL currently, so I can live with this. If a user comes with a problem around it, I'll dig a little bit more!